tests/drivers/dma/loop_transfer/boards/esp32s3_luatos_core.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_DMA_LOOP_TRANSFER_CHANNEL_NR=4
+CONFIG_DMA_LOOP_TRANSFER_SIZE=4094
```

Comment:
Any reason to use `4094` and not `4096`? Just curious.

Reply:
I copy this file from `esp32s3_devkitm.conf`, so `4094` is the number in `esp32s3_devkitm.conf`

Because this is a test program, I think that both 4096 and 4094 should run successfully